scripts/install-windows-prereqs.ps1

```diff
@@ -360,6 +360,24 @@ function Install-Python3 {
     Move-Item -Path "$tempInstallDir\*" -Destination $installDir
     Add-ToSystemPath -Path $installDir
 
+    Start-ExecuteWithRetry -ScriptBlock {
+        # Install the PIP installer
+        [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
Can use Start-FileDownload to reduce some common code

Reply:
Right.  I started with that but was having problems with it at first and once it was working, didn't want to spend time potentially breaking it since the fix might be blocking folks.   Did it work for you as is.

Reply:
I updated it to use Start-FileDownload and also made it check the hash like it does for other files downloaded

Reply:
pip issue resolve in my local machine being tested. Thanks